src/utils/discord.ts

```
@@ -0,0 +1,76 @@
+  fields?: EmbedField[];
+}
+
+export function makeEmbed(options: MakeEmbedOptions): EmbedBuilder {
+  const { title, description, color, fields } = options;
+  const embed = new EmbedBuilder();
+
+  if (title) embed.setTitle(title);
+  if (description) embed.setDescription(description);
+  if (typeof color === 'number') embed.setColor(color);
+
+  if (fields && fields.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)